renderdoc/driver/gl/wrappers/gl_emulated.cpp

```diff
@@ -1446,6 +1446,16 @@ void APIENTRY _glGetTexImage(GLenum target, GLint level, GLenum format, GLenum t
 
   size_t sliceSize = GetByteSize(width, height, 1, format, type);
 
+  bool fixBGRA = false;
+  if(!HasExt[EXT_read_format_bgra] && format == eGL_BGRA)
+  {
+    if(type == eGL_UNSIGNED_BYTE)
+      fixBGRA = true;
+    else
+      RDCERR("Can't read back texture without EXT_read_format_bgra extension (data type: %s)",
+             ToStr::Get(type).c_str());
```

Comment:
This was changed in the recent refactor, you just need `ToStr(type)` now, not `ToStr::Get(type)`.

Reply:
Sorry, I fixed it.